tools/ovc/openvino/tools/ovc/telemetry_utils.py

```diff
@@ -15,6 +15,12 @@
 except ImportError:
     import openvino.tools.ovc.telemetry_stub as tm
 
+def is_optimum():
+    import traceback
+    for frame_summary in traceback.extract_stack():
+        if "/optimum/" in frame_summary.filename:
```

Comment:
Maybe `"optimum/intel/"` for relayability?

Reply:
or "optimum/exporters/openvino"

Reply:
Are we sure that Optimum validation system don't clutter our telemetry data base?

Reply:
Optimum validation uses docker, inside docker container telemetry might be send, as docker does not pass "CI" variable.
Here is PR to fix it in optimum: https://github.com/huggingface/optimum-intel/pull/648/files